mipengine/controller/data_model_registry.py

```diff
@@ -26,16 +26,16 @@ def data_models(self):
         return self._data_models
 
     @property
-    def datasets_location(self):
+    def dataset_locations(self):
```

Comment:
The fix for `dataset_locations` should not be part of this PR. Please remove this changes completely.